### PR TITLE
Quote %epp declarations.

### PR DESCRIPTION
**Pull request by ltratt**
This updates things after https://github.com/softdevteam/grmtools/pull/46.

### CODE REVIEW

**Comment by ltratt** [comment]
After your comment in the other PR, I've added a second commit to avoid escaping. Probably should be squashed away.

File: lua5_3/lua5_3.y [hunk 1]
```diff
@@ -52,7 +52,7 @@
 %epp SEMICOL ";"
 %epp COL ":"
 %epp COMMA ","
-%epp SHORT_STR "\"<String>\""
+%epp SHORT_STR '\"<String>\"'
```
Comment (ptersilie) [inline lua5_3/lua5_3.y]: You forgot to remove the backslashes.

**Comment by ptersilie** [comment]
One minor comment. Once this is squashed and merged I rebase my PR before giving it to you for reviewing.

File: lua5_3/lua5_3.y (hunk 1 above)
Comment (ltratt) [reply lua5_3/lua5_3.y]: Oops! Fixed.

**Comment by ltratt** [comment]
If that commit works for you, should I squash?

**Comment by ptersilie** [comment]
Please squash.

**Comment by ltratt** [comment]
Squashed.

**Review by ptersilie — APPROVED** [review]

**Pull request merged**